web/controllers/users_controller.ex

```diff
@@ -0,0 +1,61 @@
+defmodule Notox.UserController do
+  use Notox.Web, :controller
+  alias Notox.User
+
+  def index(conn, _params) do
+    users = Repo.all(User)
+    render conn, :index, users: users
+  end
+
+  def show(conn, %{"id" => id}) do
+    user = Repo.get(User, id)
+    render conn, :show, user: user
+  end
+
+  def new(conn, _params) do
+    changeset = User.changeset(%User{})
+    render conn, :new, changeset: changeset
+  end
+
+  def create(conn, %{"user" => user_params}) do
+    changeset = User.registration_changeset(%User{}, user_params)
+    case Repo.insert(changeset) do
+      {:ok, user} ->
+        conn
+        |> Notox.Auth.login(user)
+        |> put_flash(:info, "User created!")
+        |> redirect(to: user_path(conn, :index))
+      {:error, changeset} ->
+        render(conn, :new, changeset: changeset)
+    end
+  end
+
+  def edit(conn, %{"id" => id}) do
```

Comment:
Thanks for the explanation. I think I get it. So I can create a `profile_controller` for stuff like editing/deleting the current user. 

Only thing I'm left wondering is if there would ever be a `profile` model? Seems like the `user` and `profile` models would encompass the same kind of stuff. More interested in the general case, I think in my case it would be excessive to create a `profile` model right now.